src/client/xmlClient.ts

```diff
@@ -103,6 +103,14 @@ export async function startLanguageClient(context: ExtensionContext, executable:
     }
   }));
 
+  const onDidGrantWorkspaceTrust = (workspace as any).onDidGrantWorkspaceTrust;
+  if (onDidGrantWorkspaceTrust !== undefined) {
+    context.subscriptions.push(onDidGrantWorkspaceTrust(() => {
+      languageClient.sendNotification(DidChangeConfigurationNotification.type, { settings: getXMLSettings(requirementsData.java_home, logfile, externalXmlSettings) });
+      workspace.getConfiguration('xml').update('downloadExternalResources.enabled', true); //set back to default setting
```

Comment:
I set it back to the initial default, but I suppose I could also store the setting in `externalXMLSettings` and retrieve once workspaceTrust is renabled?

Reply:
`externalXmlSettings` are settings other extensions might contribute, so I don't think it's a good place. I think it's fine as-is.